addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/ExternalRedirectScanRuleUnitTest.java

```diff
@@ -249,6 +317,80 @@ protected NanoHTTPD.Response serve(NanoHTTPD.IHTTPSession session) {
         assertThat(alertsRaised.get(0).getEvidence().startsWith(HttpHeader.HTTP), equalTo(true));
     }
 
+    @ParameterizedTest
+    @ValueSource(strings = {HttpFieldsNames.LOCATION, HttpFieldsNames.REFRESH})
```

Comment:
The `refresh` will not be handled properly, it does not follow the expected format (e.g. `5; url=…`).

Reply:
Thanks, I'll split these up.